### DISCUSSION TITLE
Slash commands

### DISCUSSION THREAD

**Discussion by imwezz** [hmes98318/Music-Disc#8] [Ideas]
Are in your plans replacing the prefix commands for slash commands? Or do you prefer prefix?

**Comment by hmes98318** [comment]
I plan to have both coexist.  
I like to use prefix commands,  
So the plan of slash commands is not very urgent.

**Comment by hmes98318** [comment]
Maybe we can start preparing to add slash commands in `v1.2.6`

**Reply by imwezz** [reply]
Oh got it, when you release `v1.2.5`, I'll create a draft PR

**Reply by hmes98318** [reply]
`v1.2.5` Updated
